os/arch/arm/src/imxrt/usb/host/usb_host_hci.c

```diff
@@ -0,0 +1,944 @@
+/*
```

Comment:
Please add Samsung License.

Reply:
@jeongchanKim, currently none of IMXRT arch/arm files has Samsung License header. I dont know why, so I would like to follow the same. Please suggest if you further comments on it.